pycode/memilio-generation/memilio/tools/config.json

```diff
@@ -0,0 +1,41 @@
+[
+    {
+        "source_file": "/cpp/models/ode_seir/model.cpp",
+        "path_database": "/build",
+        "namespace": "mio::oseir::",
+        "python_module_name": "test_oseir",
+        "optional": {
+            "libclang_library_path": "",
```

Comment:
This is not set but below it is. Maybe stop execution if this is not set and provide useful output message. Then we could also remove it from below. 

However, I with a path get

clang.cindex.LibclangError: cannot dynamically load position-independent executable. To provide a path to libclang use Config.set_library_path() or Config.set_library_file().

Reply:
Should be fixed.